.github/workflows/CI.yml

```diff
@@ -19,6 +19,7 @@ jobs:
         version:
           - '1.8'
           - '1.9'
+          - '1.10-nightly'
```

Comment:
Should we also do `'nightly'`? (i.e. julia main?)

Reply:
we already do, in a separate CI job so it doesn't turn the README "CI" badge red: https://github.com/JuliaTesting/ReTestItems.jl/blob/main/.github/workflows/JuliaNightly.yml